nomad/local/integration-tests.nomad

```diff
@@ -104,5 +111,235 @@ job "integration-tests" {
       }
     }
   }
+  group "analyzerlib-integration-tests" {
+    restart {
+      # Make this a one-shot job
+      attempts = 0
+    }
+
+    network {
+      mode = "bridge"
+    }
+
+    # Enable service discovery
+    service {
+      connect {
+        sidecar_service {
+          proxy {
+            upstreams {
+              # This is a hack, because IDK how to share locals across files
+              destination_name = "dgraph-alpha-0-grpc-public"
+              local_bind_port  = 9080
+            }
+          }
+        }
+      }
+    }
+
+    task "analyzerlib-integration-tests" {
+      driver = "docker"
+
+      config {
+        image      = "${var.container_registry}/grapl/analyzerlib-test:latest"
+        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
+        command    = "cd grapl_analyzerlib && py.test -v -n auto -m 'integration_test'"
+      }
+
+      env {
+        DEPLOYMENT_NAME = var.deployment_name
+        GRAPL_LOG_LEVEL = local.log_level
+        MG_ALPHAS       = "localhost:9080"
+      }
+
+      resources {
+        cpu    = 500
+        memory = 1024
+      }
+
+    }
+  }
+
+  group "analyzer-executor-integration-tests" {
+    restart {
+      # Make this a one-shot job
+      attempts = 0
+    }
+
+    network {
+      mode = "bridge"
+    }
+
+    # Enable service discovery
+    service {
+      connect {
+        sidecar_service {
+          proxy {
+            upstreams {
+              # This is a hack, because IDK how to share locals across files
+              destination_name = "dgraph-alpha-0-grpc-public"
+              local_bind_port  = 9080
+            }
+          }
+        }
+      }
+    }
+    task "analyzer-executor-integration-tests" {
+      driver = "docker"
+
+      config {
+        image      = "${var.container_registry}/grapl/analyzer-executor-test:latest"
+        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
+        command    = "cd analyzer_executor && export PYTHONPATH=\"$(pwd)/src\"; py.test -n auto -m 'integration_test'"
+      }
+
+      env {
+        # aws vars
+        AWS_REGION                  = var.aws_region
+        GRAPL_AWS_ENDPOINT          = var.aws_endpoint
+        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
+        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+
+        GRAPL_LOG_LEVEL = local.log_level
+
+        # We don't appear to need this set.
+        # TODO double-check why
+        GRAPL_ANALYZER_MATCHED_SUBGRAPHS_BUCKET = ""
```

Comment:
@wimax-grapl do you know off the top of your head if these environment variables need to be defined at all? for these tests?

Reply:
We probably do need them to be defined; I see it in use inside of Analyzer Executor (not necessarily the test) at:
```
            for exec_hit in self.poll_process(rx=rx, analyzer_name=analyzer_name):
                with self.metric_reporter.histogram_ctx(
                    "analyzer-executor.emit_event.ms",
                    (TagPair("analyzer_name", exec_hit.analyzer_name),),
                ):
                    emit_event(self.analyzer_matched_subgraphs_bucket, s3, exec_hit)
```

basically, emitting to S3 the next step in the pipeline